examples/plot_mne_example.py

```diff
@@ -0,0 +1,223 @@
+
+###################################################################################################
+
+# Explicitly adding a default EEG average reference required for source localization
+raw.set_eeg_reference()
+
+###################################################################################################
+
+# Explore the data
+raw.info()
+
+###################################################################################################
+
+# Plot the channel locations and labels
+raw.plot_sensors(show_names=True)
+
+###################################################################################################
+
+# Creating epochs
+reject = dict(eeg=180e-6)
+event_id, tmin, tmax = {'left/auditory': 1}, -0.2, 0.5
+events = mne.read_events(event_fname)
+epochs = mne.Epochs(raw, events=events, event_id=event_id,
+				tmin=5, tmax=125, baseline=None, preload=True)
+
+###################################################################################################
+
+# Creating Power Spectra Densities
+psd, freqs = mne.time_frequency.psd_welch(epochs, fmin=1., fmax=50., n_fft=2000, n_overlap=250, n_per_seg=500)
+
+###################################################################################################
+# fooofgroup
+# ------------------
+#
+# The FOOOFGroup object is used to fit
+# FOOOF models across the power spectra. A list of FOOOFGroup objects is returned.
+#
+
+###################################################################################################
+
+# Initialize a FOOOF group object, with desired settings
+fg = FOOOFGroup(peak_width_limits=[1, 6], min_peak_amplitude=0.075,
+                    max_n_peaks=6, peak_threshold=1, verbose=False)
+
+###################################################################################################
+
+# Shaping the PSDs so that we can run fooofgroup
+fooof_psd = np.squeeze(psd[4,:,:]) 
+n_channels, n_freq = fooof_psd.shape
+num_blocks = len(mne.read_events(event_fname))
+
+# Setting frequency range
+freq_range = [3, 32]
+
+# This returns a list of FOOOFGRoup objects
+fg.fit(freqs, fooof_psd, freq_range)
+fg.plot()
+
+###################################################################################################
+
+# Define bands of interest
+bands = {'theta': [2,7],
+         'alpha': [8,14],
+         'beta': [15,30]}
+
+# Creating dictionaries to store all the aperiodic properties across frequencies 
+results = {}
+for band_name in bands.keys():
+    results[band_name] = np.zeros(shape=[ num_blocks, n_channels, n_feats])
+
+# Creating dictionaries to store all the periodic properties across frequencies 
+slope_results = np.zeros(shape=[num_blocks, n_channels, 2])
+
+###################################################################################################
+
+# Populating periodic and aperiodic values 
+for block in range(0, num_blocks):
+    for ind, res in enumerate(fg):
+        slope_results[block, ind, :] = res.background_params
```

Comment:
We've updated FOOOF related terminology a bit. What we used to call 'slope' is now called 'aperiodic exponent', or just 'exponent'. Update all references to 'slope' to 'exponent'.